### DISCUSSION TITLE
How to make code size bigger ?

### DISCUSSION THREAD

**Discussion by DelinQu** [mmistakes/minimal-mistakes#3260] [Q&A] [answered]
I wrote a piece of code in `md` and post it to my `blog posts`. It looks nice but the code is a bit too small.

How can i make it bigger ?

![image](https://user-images.githubusercontent.com/60593268/144164305-f3cc54c4-0944-4263-9a73-f8a974a07a58.png)

Thank you in advance ! :smile:

**Comment by mmistakes** [comment, marked as answer]
Looks like you're using a fork of the Minimal Mistakes theme called Academic Pages. That's not exactly current with this theme. If you compare the code blocks against MM, you'll notice they're much smaller and styled differently.

MM's are much improved so if you can switch to it instead of using the fork you'd the improved styling. Otherwise you'll need to be comfortable with CSS and modify it directly... you can do most of it by bumping up the `font-size` in [`_syntax.scss`](https://github.com/DelinQu/DelinQu.github.io/blob/master/_sass/_syntax.scss#L31).

Just changing the line above to:

```scss
font-size: 1em;
```

Will dramatically increase the size and look something like this:

![Screen Shot 2021-12-01 at 9 19 48 AM](https://user-images.githubusercontent.com/1376749/144250853-0679ce6b-0f6b-43d7-90b8-46ffa5b31a38.png)

**Reply by DelinQu** [reply]
Thanks so much ! I will switch to MM:laughing:
